src/Logging/Logging.Abstractions/src/EventId.cs

```diff
@@ -70,7 +70,7 @@ public override string ToString()
         /// Indicates whether the current object is equal to another object of the same type. Two events are equal if they have the same id.
         /// </summary>
         /// <param name="other">An object to compare with this object.</param>
-        /// <returns><code>true</code> if the current object is equal to the other parameter; otherwise, <code>false</code>.</returns>
+        /// <returns><c>true</c> if the current object is equal to the other parameter; otherwise, <c>false</c>.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true" /> if the current object is equal to the other parameter; otherwise, <see langword="false" />.</returns>
```